### PR TITLE
Check for actions

### PR DESCRIPTION
**Pull request by apereverzin**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-283

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java [hunk 1]
```diff
@@ -29,7 +29,10 @@ public void should_create_and_update_case_automatically()
         assertCompletedProcessorResult(zipArchiveCreate.fileName, "CASE_CREATED");
 
         String ccdId = getZipFileStatus(zipArchiveCreate.fileName).get().ccdId;
-        assertCaseFields(ccdId, "Name", "Surname", "e2e@test.dev");
+        Map<String, Object> caseDataCreated =
+            CcdClient.getCaseData(ccdId, BULKSCAN_AUTO.idamUserName, BULKSCAN_AUTO.idamPassword);
+        assertCaseFields(caseDataCreated, "Name", "Surname", "e2e@test.dev");
+        assertCaseEnvelopes(caseDataCreated, new String[]{"create"});
```
Comment (mdayican) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java:35]: can we check `id` is not null

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java [hunk 2]
```diff
@@ -45,7 +48,10 @@ public void should_create_and_update_case_automatically()
 
         assertCompletedProcessorResult(zipArchiveUpdate.fileName, "AUTO_UPDATED_CASE");
 
-        assertCaseFields(ccdId, "Name1", "Surname1", "e2e1@test.dev");
+        Map<String, Object> caseDataUpdated =
+            CcdClient.getCaseData(ccdId, BULKSCAN_AUTO.idamUserName, BULKSCAN_AUTO.idamPassword);
+        assertCaseFields(caseDataUpdated, "Name1", "Surname1", "e2e1@test.dev");
+        assertCaseEnvelopes(caseDataUpdated, new String[]{"create", "update"});
```
Comment (mdayican) [inline src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java:54]: can we check `id` is not null

Comment (apereverzin) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java]: Done

File: src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java (hunk 1 above)
Comment (apereverzin) [reply src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/BulkScanAutoUpdateTest.java]: Done @mdayican

**Review by mdayican — APPROVED** [review]

**Pull request merged**